### PR TITLE
Add --version option to show abi3audit version information

### PR DESCRIPTION
**Pull request by wsfulton**
Current output for new option in a commonly used output format of program name followed by version number:
```
$ python -m abi3audit --version
abi3audit 0.0.19
```

### CODE REVIEW

File: abi3audit/_cli.py [hunk 1]
```diff
@@ -175,55 +175,72 @@ def main() -> None:
...
-    parser.add_argument(
+    version_group = parser.add_argument_group()
+    version_group.add_argument(
+        "-V",
+        "--version",
+        action="store_true",
+        help="show abi3audit version information",
+    )
+    main_group = parser.add_argument_group("main arguments")
+    main_group.add_argument(
```
Comment (woodruffw) [inline abi3audit/_cli.py:186]: I think this makes the parser a lot harder to follow -- could we use `action='version'` instead? That should handle everything automatically and avoid the need for separate arg groups.

For example:

```python
parser.add_argument("-V", "--version", action='version', version=f"%(prog)s {__version__}")
```

**Comment by woodruffw** [comment]
Thanks @wsfulton! This is great to have, but I think we can do it with a one-line change instead of parser groups. See my suggestion above 🙂

File: abi3audit/_cli.py (hunk 1 above)
Comment (wsfulton) [reply abi3audit/_cli.py]: Oh that's much better, I didn't know that argparse had built in support for version display.

**Review by woodruffw — APPROVED** [review]
Thanks, LGTM! I can deal with the README CI failure in a follow-up.

**Pull request merged**